frontend/app/routes/application.js

```diff
@@ -0,0 +1,31 @@
+import Route from "@ember/routing/route";
+import { inject as service } from "@ember/service";
+import ApplicationRouteMixin from "ember-simple-auth/mixins/application-route-mixin";
+
+export default Route.extend(ApplicationRouteMixin, {
+  /**
+   * The session service
+   *
+   * @property {EmberSimpleAuth.SessionService} session
+   * @public
+   */
+  session: service("session"),
+
+  /**
+   * The actions for the application route
+   *
+   * @property {Object} actions
+   * @public
+   */
+  actions: {
+    /**
+     * Invalidate the session
+     *
+     * @method invalidateSession
+     * @public
+     */
+    invalidateSession() {
+      this.get("session").invalidate();
+    }
```

Comment:
Since you have a `logout` action in the controller you don't need this. Also, remove the useless comments